src/helpers/cellHotspots.js

```diff
@@ -0,0 +1,40 @@
+const qs = require('qs')
+const fetch = require('node-fetch')
+
+const baseUrl = process.env.CELL_HOTSPOT_BASE_URL
+const apiKey = process.env.CELL_HOTSPOT_API_TOKEN
+
+const makeRequest = async (route, params) => {
+  const urlRoute = [baseUrl, route].join('/')
+  let url = urlRoute
+  if (params) {
+    url = [urlRoute, qs.stringify(params)].join('?')
+  }
+
+  const response = await fetch(url, {
+    method: 'GET',
+    headers: {
+      'Authorization': `Bearer ${apiKey}`,
+      'Cache-Control': 'no-cache',
+      'Content-Type': 'application/json',
+    },
+  })
+
+  if (!response.ok) {
+    throw { status: response.status, message: response.statusText }
+  }
+
+  const text = await response.text()
+  try {
+    const json = JSON.parse(text)
+    return json.data || json
+  } catch (err) {
+    return text
+  }
+}
+
+export const getLastHeartbeat = async (gatewayAddress) =>
```

Comment:
Just a little nitpick... If you aren't doing anything with the result of `makeRequest` there's not need to `await`, you can just return the promise

```ts
export const getLastHeartbeat = (gatewayAddress) =>
  makeRequest(`heartbeats/hotspots/${gatewayAddress}/last`)
  ```

Reply:
updated